test/ci-operator-configresolver-integration/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# This test runs the ci-operator-configresolver and verifies that it returns
+# properly resolved configs
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+ROOTDIR=$(pwd)
+WORKDIR="$( mktemp -d )"
+trap "rm -rf ${WORKDIR}" EXIT
+
+pushd $WORKDIR
+go build -o tester $ROOTDIR/test/ci-operator-configresolver-integration/main.go
+# copy registry to tmpdir to allow tester to modify registry
+cp -a $ROOTDIR/test/ci-operator-configresolver-integration/ tests
+ci-operator-configresolver -config tests/configs -log-level debug &
+PID=$!
+# wait for registry to be resolved
+sleep 1
+if ! output=$( ./tester -serverAddress "http://127.0.0.1:8080" ); then
```

Comment:
Use the `ci-operator-configresolver` command directly. This is going to be resolved in the `src` image when the integration tests will run.

Reply:
This was for the tester binary, not the configresolver. I'm removing the tester binary though, so this won't be necessary.